backend/api/migrations/0002_auto_20170310_2142.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-10 21:42
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import csv
+
+def load_initial_data(apps, schema_editor):
+    Business = apps.get_model("api", "Business")
+    Inspection = apps.get_model("api", "Inspection")
+
+    with open('../../../DOHMH_NYC_Restaurant_Inspection_Results.csv') as csv_file:
+        reader = csv.reader(csv_file, delimiter=',')
+        header = next(reader)
+        for row in reader:
```

Comment:
So i got back into my code and what i'm having trouble figuring out is how to check for and create the list of unique business objects. If i iterate over all the rows and create a set of all the camis values i know the length the business list should be, but my brain isn't translating that to how to get that row if it meets that condition.

My first flawed logic was doing something like:
![image](https://cloud.githubusercontent.com/assets/7889075/24927479/2a9c8b9c-1ecd-11e7-95e1-86b3c0b8f3a4.png)

i know i'm thinking about this wrong, but not coming up with the right way.